### ISSUE TITLE
Enhance docs before release

### ISSUE THREAD

**Issue by aguschin** [iterative/mlem#270]
After closing #151, this is a follow-up ticket with what's currently in the work. Ideally, most of this should be done before the 1st of June.

- https://github.com/iterative/mlem.ai/issues/70
- https://github.com/iterative/mlem.ai/issues/75
- https://github.com/iterative/mlem.ai/issues/63
- https://github.com/iterative/mlem.ai/issues/58
- https://github.com/iterative/mlem.ai/pull/79
- https://github.com/iterative/mlem.ai/pull/68
- https://github.com/iterative/mlem.ai/pull/67
